app/api/ApplicationApi.js

```diff
@@ -711,7 +711,7 @@ const ApplicationApi = {
             {
                 fee: {
                     amount: 0,
-                    asset_id: objects.feeAsset.get("id")
+                    asset_id: accountUtils.getFinalFeeAsset(from,"withdraw_permission_claim")
```

Comment:
I think this should be `to`.
The page to test it is Hamburg menu -> Spotlight -> Direct debit.

Reply:
done, thanks.